.github/workflows/check-pull-request.yml

```diff
@@ -83,3 +83,14 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+
+  test-journey:
+    name: Run Journey Tests
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Could we make this conditional on the previous check with `needs: pr-validator`?

It'll slow down the checks a bit, but it will mean that we don't sun resources unnecessarily